src/Lumi/Components/Form/Table.purs

```diff
@@ -270,3 +276,10 @@ column label orientation (FormBuilder f) =
             { className = Nullable.notNull "labeled-field--validation-warning"
             , children = [ R.text w ]
             }
+
+-- | Make the props available, for convenience.
```

Comment:
If we didn't export this function, it wouldn't be possible to have the columns of a TableFormBuilder depend on the props, right? If that's the case I don't think it's quite right to describe this function as being for convenience; to me, describing something as "for convenience" means that it's possible to construct from the rest of the public API.

Reply:
Yes, that's a good take. I'll update this comment and the ones in `Lumi.Components.Form`. Thanks!